README.md

```diff
@@ -1,2 +1,4 @@
 # underdog-devs-docs
 Documentation for the Underdog Devs Labs project
```

Comment:
We appreciate you working with Web to ensure that our changes made wouldn't affect the DS API!